src/templates/uploads.html

```diff
@@ -3,7 +3,7 @@
   <div class='modal-dialog modal-lg' role='document'>
     <div class='modal-content'>
       <div class='modal-header'>
-        <h5 class='modal-title' id='plainTextModalLabel'></h5>
+        <h5 class='modal-title' id='plainTextModalLabel'>placeholder</h5>
```

Comment:
what's this? What happens to "placeholder" string?

Reply:
The plain text modal heading is always replaced with the file name via js when a user wants to look at the file. However, an empty heading is not valid html, so "placeholder" was added. But this could be any other text.

Reply:
I see, then you can add a comment on top of that line explaining that ;)